spec/order_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'order'
+require 'menu'
+
+describe Order do
```

Comment:
Please could you isolate the `Order` tests by using a test double for `Menu`? For example:
```ruby
subject { described_class.new(menu) }
```
You'll need to define `menu` as a double that allows the `dishes` method.